src/microbe_stage/organelle_components/SlimeJetComponent.cs

```diff
@@ -7,8 +7,8 @@
 /// </summary>
 public class SlimeJetComponent : IOrganelleComponent
 {
+    public const string MUCOCYST = "mucocyst";
```

Comment:
As this is now public I think it is even *more* important for this to have a clear name so could you please rename this bac kto MUCOCYST_UPGRADE_NAME?